client/src/components/HomeScreen.tsx

```diff
@@ -138,17 +138,19 @@ const ListAuditsAuditAdmin: React.FC = () => {
               You haven&apos;t created any audits yet for {organization.name}
             </p>
           ) : (
-            organization.elections.map(election => (
-              <AuditLink
-                key={election.id}
-                to={`/election/${election.id}`}
-                intent="primary"
-                large
-                fill
-              >
-                {election.auditName}
-              </AuditLink>
-            ))
+            organization.elections
+              .sort((a, b) => (a.auditName > b.auditName ? 1 : -1))
```

Comment:
two things:
- do you think we should sort the organizations as well?
- i think i used `sortBy` below - do you want to use that here? i think the reason i did that was because `sort` modifies the array in place, so it's generally a bit dangerous to use. and also so you don't have to define your own comparator function. but i don't think the in-place modification is a big risk here, so maybe doesn't matter

Reply:
@jonahkagan Oh yes, I'm tracking what you did there now. Sure, no problem. I swapped to `sortBy` and sorted both organizations and audits now.